### ISSUE TITLE
Incorrect vent order not flagged as an error in CEdit

### ISSUE THREAD

**Issue by rpeacoc** [firemodels/cfast#1282]
The attached file should generate error messages because the order of the compartments in the vent specification is backwards (lower-numbered room should always be first).

```
&HEAD VERSION = 7300, TITLE = 'CFAST Simulation' /
 
!! Scenario Configuration 
&TIME SIMULATION = 900 PRINT = 50 SMOKEVIEW = 10 SPREADSHEET = 10 / 
&INIT PRESSURE = 101325 RELATIVE_HUMIDITY = 50 INTERIOR_TEMPERATURE = 20 EXTERIOR_TEMPERATURE = 20 /
&MISC  LOWER_OXYGEN_LIMIT = 0.15 / 
 
!! Compartments 
&COMP ID = 'Comp 1'
      DEPTH = 2.4 HEIGHT = 2.4 WIDTH = 3.6 CEILING_MATL_ID = 'OFF' WALL_MATL_ID = 'OFF' FLOOR_MATL_ID = 'OFF'
      ORIGIN = 0, 0, 0 GRID = 50, 50, 50 /
&COMP ID = 'Comp 2'
      DEPTH = 2.4 HEIGHT = 2.4 WIDTH = 3.6 CEILING_MATL_ID = 'OFF' WALL_MATL_ID = 'OFF' FLOOR_MATL_ID = 'OFF'
      ORIGIN = 0, 0, 0 GRID = 50, 50, 50 /
 
!! Wall Vents
&VENT TYPE = 'WALL' ID = 'WallVent_1' COMP_IDS = 'Comp 2', 'Comp 1' TOP = 2, BOTTOM = 0, WIDTH = 1
  FACE = 'FRONT' OFFSET = 1.8 /
&VENT TYPE = 'WALL' ID = 'WallVent_2' COMP_IDS = 'OUTSIDE', 'Comp 1' TOP = 2, BOTTOM = 0, WIDTH = 1
  FACE = 'FRONT' OFFSET = 0 /

```

**Comment by rpeacoc** [comment]
Need to check and flag/correct both inside vent specifications (i.e., 3 --> 2) and outside to inside (i.e., #rooms+1 --> 1 since outside  is numbered as one plus the number of rooms in the simulation).  We should be able to just fix it when its input by the user

**Comment by rpeacoc** [comment]
Fixed in 0c956d3
